packages/testing/package.json

```diff
@@ -20,7 +23,7 @@
     "@firebase/logger": "0.2.1",
     "@firebase/util": "0.2.44",
     "@types/request": "2.48.4",
-    "grpc": "1.24.2",
+    "firebase": "7.13.1",
```

Comment:
It is redundant. `firebase` is already a dependency.
Was grpc a redundant dependency anyway regardless of this change?